analyses/telomerase-activity-prediction/RUN-telomerase-activity-prediction.sh

```diff
@@ -21,33 +21,38 @@ if [ "${RUN_FOR_FIGURES}" == "1" ]; then
   #generate telomerase activities using gene expression data from collapse RNA seq data files
   Rscript --vanilla 01-run-EXTEND.R --input ../../data/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds --output results/TelomeraseScores_PTBAStranded_FPKM.txt
 
-else 
+  # Generate again, but this time with tumor purity thresholding
+  Rscript --vanilla 01-run-EXTEND.R --apply_tumor_purity_threshold --input ../../data/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds --output results/TelomeraseScores_PTBAStranded_FPKM_thresholded.txt
 
-  #generate collapsed data for count files 
+  # Explore thresholded results
+  Rscript --vanilla -e "rmarkdown::render('07_EXTEND-at-threshold.Rmd')"
+
+else
```

Comment:
Off the top of my head, I don't really know why we'd expect this to be run when we're running things for figures. Can you tell me about your thought process here? Related (at least in my mind): There's no corresponding change in `analyses/telomerase-activity-prediction/README.md`.

Reply:
Running for figures, in my mind, is equivalent to "manuscript-related tasks" and re-running with the filter like one of those situations to me!

Reply:
I was going to say wouldn't we want to run it to make sure all analyses in the manuscript are up to date _or_ when we run for figures but then I noticed that we use this flag when we re-run all analyses included in the manuscript: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/059047a19025f4afd9143fceff174e6da3392747/scripts/run-manuscript-analyses.sh#L87

That strikes me as a little bit weird, but okay, not a hill I want to die on.

Reply:
> That strikes me as a little bit weird, but okay, not a hill I want to die on.

We agree!